docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -27,7 +27,7 @@ This article assumes that you know how to use a command line and have a preferre
 1. Open a Command Line/Terminal.
```

Comment:
you didn't change this but style guide says that "Describe entering commands at a command prompt, not on a command line.", so probably this should be changed to Open a command prompt/terminal.